### PR TITLE
[docs] Added voyager 1.8.4 docs

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/yugabyte-voyager/migrate/migrate-steps/

### CODE REVIEW

File: docs/content/preview/yugabyte-voyager/known-issues/postgresql.md [hunk 1]
```diff
@@ -1046,7 +1046,9 @@ CREATE TABLE public.locations (
 
 **GitHub**: [Issue 1731](https://github.com/yugabyte/yb-voyager/issues/1731)
 
-**Description**: If your source datatabase has any of these datatypes on the columns - `POINT`, `LINE`, `LSEG`, `BOX`, `PATH`, `POLYGON`, or `CIRCLE`, it will be skipped during the migration.
+**Description**: For live migration, if your source datatabase has any of these datatypes on the columns - `POINT`, `LINE`, `LSEG`, `BOX`, `PATH`, `POLYGON`, or `CIRCLE`, it will be skipped during the migration.
```
Comment (sanyamsinghal) [inline docs/content/preview/yugabyte-voyager/known-issues/postgresql.md:1049]: typo `datatabase`

File: docs/content/preview/yugabyte-voyager/migrate/assess-migration.md [hunk 2]
```diff
@@ -23,14 +24,21 @@ When you run an assessment, Voyager gathers key metadata and metrics from the so
...
 - **Performance metrics:** Analyzes workload characteristics to recommend optimizations in YugabyteDB.
 - **Migration time estimate:** Provides an estimated time for data import into YugabyteDB based on experimental data.
+- **Unsupported query constructs:** Identifies SQL features and constructs not supported by YugabyteDB, such as advisory locks, system columns and XML functions, and provides a list of queries containing these constructs.
 
 {{< warning title="Note" >}}
-The recommendations are based on testing using a [RF3](../../../architecture/docdb-replication/replication/#replication-factor) YugabyteDB cluster on instance types with 4GiB memory per core and running v2024.1.
+
+- The recommendations are based on testing using a [RF3](../../../architecture/docdb-replication/replication/#replication-factor) YugabyteDB cluster on instance types with 4GiB memory per core and running v2024.1.
+
+- For unsupported query construct detection, ensure [pg_stat_statements extension](../../explore/ysql-language-features/pg-extensions/extension-pgstatstatements/) is properly installed and enabled on source.
```
Comment (sanyamsinghal) [inline docs/content/preview/yugabyte-voyager/migrate/assess-migration.md:38]: tried clicking on pgstatstatements hyperlink. Seems broken.

**Review by sanyamsinghal — APPROVED** [review]

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 3]
```diff
@@ -13,6 +13,30 @@ type: docs
...
+
+### Enhancements
+
+- Added support in [assess-migration](../reference/assess-migration/) and [analyze-schema](../reference/schema-migration/analyze-schema/) command to report the unsupported datatypes for live migration during fall-forward/fall-back such as user-defined types, array of enums, and so on.
+- Improved coverage of reporting PostGIS datatypes by including BOX2D, BOX3D, and TOPOGEOMETRY in the assess-migration and analyze-schema report.
+- Included Voyager version details in assess-migration and analyze-schema reports.
+- Added a confirmation prompt (yes/no) for start clean during data export, allowing users to confirm before proceeding.
+- Enhanced export and import commands output to display exported/imported table list by row counts, with the largest tables on top.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:32]:
```suggestion
- Enhanced export and import command output to display exported/imported table list by row count, with the largest tables first.
```

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 4]
```diff
@@ -13,6 +13,30 @@ type: docs
...
+
+### Bug fixes
+
+- Fixed [import data](../reference/data-migration/import-data/) and `import data file` commands to ensure batches created from data files don't exceed the default batch size limit (200MB), preventing "RPC message too long" errors with large rows. Also, added an immediate check to error out for single row size over 200MB.
+- Fixed an issue in `analyze-schema` where partitioned tables were incorrectly reported as having the insufficient columns in the primary key constraint due to a regex misidentifying the CONSTRAINT clause before primary key definition.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:37]:
```suggestion
- Fixed an issue in `analyze-schema` where partitioned tables were incorrectly reported as having insufficient columns in the primary key constraint due to a regex misidentifying the CONSTRAINT clause before primary key definition.
```

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 5]
```diff
@@ -13,6 +13,30 @@ type: docs
...
+- Fixed the object names in the schema summary of `assess-migration` and `analyze-schema` reports for OPERATOR names and to uniquely identify the INDEX, TRIGGER, and POLICY objects by adding table name to it.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:38]: not quite sure how to fix this. the operator names were wrong? add table name to what?

Comment (sanyamsinghal) [reply docs/content/preview/yugabyte-voyager/release-notes.md:38]: cc @priyanshi-yb

Comment (priyanshi-yb) [reply docs/content/preview/yugabyte-voyager/release-notes.md:38]: for OPERATOR objects, we fixed the names as they were not correctly shown earlier but for INDEX TRIGGER and POLICY ones we renamed them in the reports to also include table names for unique identification in their object names with the format `<object_name> ON <table_name>`.

**Pull request merged**